Counter/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Counter</title>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <header></header>
+    <main>
+      <div class="counter_background">
+        <div class="counter">
+          <button type="button" onClick="dec3()">
+            <a>-3</a>
+          </button>
```

Comment:
```suggestion
          <button type="button" onClick="dec3()">
            -3
          </button>
```
There is no need in `a`